opencog/nlp/relex2logic/gender.scm

```diff
@@ -0,0 +1,49 @@
+(define gender
+	(BindLink
+		(VariableList
+			(TypedVariableLink
+				(VariableNode "$a-parse")
+				(TypeNode "ParseNode")
+			)
+			(TypedVariableLink
+				(VariableNode "$word")
+				(TypeNode "WordInstanceNode")
+			)	
+			(TypedVariableLink
+				(VariableNode "$gtype")
+				(TypeNode "DefinedLinguisticConceptNode")
+			)
+		)
+		(AndLink
+			(WordInstanceLink
+				(VariableNode "$word")
+				(VariableNode "$a-parse")
+			)
+			(InheritanceLink 
+			    (VariableNode "$word")
+				(VariableNode "$gtype")
+				
+			)
+			(InheritanceLink 
+			    (VariableNode "$word")
+				(DefinedLinguisticConceptNode "person")
+				
+			)
+		)
+   (ListLink
+		(ExecutionOutputLink
+			(GroundedSchemaNode "scm: pre-gender-rule")
+			(ListLink
+				(VariableNode "$word")
+				(VariableNode "$gtype")
+			)
+		)
+   )
+))
+
+(define (pre-gender-rule word gtype)
+ (cond  ((or (string=? (cog-name gtype) "masculine") (string=? (cog-name gtype) "feminine"))	 

```

Comment:
the gender-rule does this checks, so why have it here? Are you thinking of having another rule for `person` gender-type?